App/tools/config

```diff
@@ -0,0 +1,5 @@
+DEFAULT_EPICS_VERSIONS = 7.0.7
+EPICS_LOCATION = /opt/epics
+GIT_DOMAIN = lightsource.ca
+BUILDCLASSES = Linux
+EXCLUDE_ARCHS = clang
```

Comment:
I do not want to have the config file in the public repo. It is very facility dependent and may change often.